.golangci.yml

```diff
@@ -73,6 +73,8 @@ issues:
     - path: test/e2e/
       linters:
         - wsl
+        - gocognit
+        - gocyclo
```

Comment:
Are you adding or removing these linters?

Reply:
Those linters are excluded for files under "test/e2e/". So removing for tests.

Reply:
for tests it makes sense, thanks.